doc/Options/z390_filetypes.md

```diff
@@ -0,0 +1,32 @@
+# z390 File types
+
+The scope value details where the option is used.
+
+* M - mz390 macro processor which reads ASCII assembler source and generates expanded BAL
+* A - az390 assembler which reads BAL and generated OBJ relocatible object code
+* L - lz390 linker which reads OBJ relocatible object files and generates 390 load module
+* E - ez390 emulator runtime which loads and executes 390 load module
+
+File extension | Scope | Format | File Description | File or Report Format Description
+---|---|---|---|---
+BAL | MA | ASCII | Basic Assembly Language generated ASCII source file generated by mz390 when option BAL is specified.  When option ASM is on (the default), mz390 starts az390 as a separate process running in parallel and passes generated BAL statements directly to it.  This allows the first pass of assembler processing to be overlapped with macro expansion and eliminates need to write this file to external disk.|Basic Assembler format is free form within first 71 characters with optional label starting in position 1 followed by opcode followed by operands if any.  If continuation is required a non-space character appears in position 72.  Positions 73-80 may be used for sequence numbering.  The ICTL  instruction may be used to change these default setting but it is not recommended as the macro and copybook files are all in the default format.  Embedded AREAD records longer than 71 characters must be in standard continuation format for preloading of MLC and MAC files (a z390 restriction).  Note the default NOALLOW limits AREAD and PUNCH records to maximum of 80.
```

Comment:
Sentence beginning "The ICTL ...". "... these default setting but ..." -> "... these default settings but ...".

Reply:
Resolved by 52df1cf